src/Apps/ArtQuiz/ArtQuizInterface/ArtQuizButtons.tsx

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
Ditto here. We use a `Components`, folder throughout force for these types of things, rather than a unique-to-the-app and one-off `ArtQuizInterface` folder. If we wanted to refactor the app later at scale (for example, moving all `Components` foders to another location), this singleton pattern would be lost in find and replace. 

Worth giving a good read over our whole [best practices](https://github.com/artsy/force/blob/main/docs/best_practices.md#keep-file-structure-organized) doc.

Reply:
No problem! And totally understand how polish works. A carpenter sets up his project, and slowly and steadily refines things until they're final. And usually that kind of work happens right at the end. 

Just noting that _minor_ requests like this can be taken care of nearly instantly (right click -> folder rename), versus needing to keep track of comment after comment from collaborators through time, which is how stuff slips through the cracks. If the comment or request for change were more significant -- ie, "The use of context here is can be done entirely differently!", to which y'all respond "this is still in progress, and will develop further and likely change over time!" that's an entirely different matter, and totally understandable. Future refactoring will likely happen (as is the nature of the work), and things will change. But _minor things_ like the above are better addressed in the moment, or immediately in a follow-up, so they don't get lost, especially because they can be addressed so quickly.